app/models/ark.rb

```diff
@@ -17,6 +17,13 @@ def self.find(ezid)
     nil
   end
 
+  def self.update(ezid, new_url)
+    return if ezid.start_with?(EZID_TEST_SHOULDER)
+    identifier = Ezid::Identifier.find(ezid)
+    identifier.target = new_url
+    identifier.save!
```

Comment:
Notice that these lines are not tested. I [added a ticket](https://github.com/pulibrary/pdc_describe/issues/146) to add a separate test for this functionality that focuses on the Ark class itself instead of testing this as part of the Work class (as we do today.)